### PR TITLE
Popup menu

### PR DESCRIPTION
**Pull request by Wils-FOURR-speed**
This part of the project was accomplished using the following:
- html, css and js files
- Html/css/js linters
- Media links and live demo referral links

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -406,6 +407,129 @@ footer {
...
+}
+
+.dclosebtn {
+  margin: 0 0 8px auto;
+  display: flex;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
```
Comment (rloterh) [inline style.css]: - [ ] Some portions of the popup window are not showing on the top and bottom sides on the mobile screen sizes. Let's ensure to improve upon this aspects accordingly.

**Your design:**        |  **The original design:**
:-------------------------:|:-------------------------:
Home Page | Home Page

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Wilsan, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!
🏁 🚦 🏃 🌴 🏫 🌳 🏘️

## Highlights 👍  
- Popup displays and works correctly on desktop version!
- Objects correctly loaded dynamically using JavaScript!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
Let's improve upon the look of the ReadMe by removing the underlined aspect:

image

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -87,108 +87,49 @@ <h1 class="work">
...
-                        See Project
-                    </button>
+            <div class="flex spaceb">
+                <h2 id="tit">
+                    Keeping track of hundreds of <br>
+                    components
+                </h2>
+                <div class="flexBtn">
+                    <a href="https://wils-fourr-speed.github.io/My-portfolio/" class="dbtn">see live <img src="live.svg" alt=""></a>
+                    <a href="https://github.com/Wils-FOURR-speed/My-portfolio" class="dbtn">see source <img src="github.svg" alt=""></a>
```
Comment (divinecharlotte) [inline index.html:102]: - [x] As requested by the previous reviewer kindly consider making sure that these buttons are also visible on your mobile pop-up to make sure that your website is interactive you can do so by adding a scroll on the pop-up so if the content is more there will be a way to scroll up and down in the popup
Screen Shot 2023-06-24 at 23 34 38

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @Wils-FOURR-speed   , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Branch named well👍 
- Descriptive readme 👍 

 




 

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @Wils-FOURR-speed 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**